lxc/cluster_group: Added lxc cluster group add (#11980)
canonical/lxd: doc/howto/cluster_groups.md

    lxc cluster group assign server1 default,gpu

To add a cluster member to a specific group without removing it from other groups, use the `lxc cluster group add` command.

> For example, to add `server1` to the `gpu` group and also keep it in the `default` group, use the following command:

Reviewer: Ah I see you've made the requested changes as new commit, but please can you rebase and squash/edit your earlier commit so that we dont see the evolution of the particular change in the final PR.

    lxc cluster group assign server1 default,gpu

> To add a cluster member to a specific group, use `lxc cluster group add` command.
> This command adds a cluster member to the specified group.

Reviewer 2: ```suggestion
To add a cluster member to a specific group without removing it from other groups, use the `lxc cluster group add` command.
```

> For example, to add `cluster:node1` to the `newgroup` group , use the following command:

Reviewer: Please can you use `server1` rather than `cluster:node1` to be consistent and also to avoid using "node" terminology we dont use in user face docs/messages. Thanks

>     lxc cluster group add cluster:node1 newgroup

Reviewer 2: ```suggestion
    lxc cluster group add server1 gpu
```

Author: Hi, @ru-fu made the changes as requested. Thanks

>     lxc cluster group add cluster:node1 newgroup

Reviewer: Please use "server1" rather than "cluster:node1"
